.github/workflows/ci.yml

```
@@ -123,6 +125,9 @@ jobs:
           # See https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html
           UBSAN_OPTIONS: print_stacktrace=1
 
+      - name: Benchmarks
+        run: cmake --build ./build --config Release --verbose --target run_compiler_benchmark
```

```suggestion
        run: cmake --build ./build --config Release --verbose --target benchmark
```

We can keep it simple? I think we will expand on more benchmarks than just the compiler at some point. For example, we might end up profiling the JSON class as a consequence

---

Yeah is simpler Indeed :)

I do back and forth between both approach:
- simplicity on ones side
- specificity on the other as you rarely want to run all locally, and even on the CI, you'll prolly look for a specific segment..